pkg/broker/broker.go

```diff
@@ -53,9 +54,46 @@ func NewAnsibleBroker(
 		engine:        &engine,
 	}
 
+	// If no openshift target is provided, assume we are running in an openshift
+	// cluster and try to log in using mounted cert and token
+	inCluster := clusterConfig.Target == ""
+	if inCluster {
+		err := broker.Login()
+		if err != nil {
+			return broker, err
+		}
+	}
+
 	return broker, nil
 }
 
+func (a AnsibleBroker) Login() error {
```

Comment:
@rthallisey took a stab at it, kind of feel like the OcLogin function should live somewhere else but not sure where that would be...